src/Serilog.Sinks.PeriodicBatching/Serilog.Sinks.PeriodicBatching.csproj

```diff
@@ -2,9 +2,10 @@
 
   <PropertyGroup>
     <Description>Buffer batches of log events to be flushed asynchronously.</Description>
-    <VersionPrefix>3.1.1</VersionPrefix>
+    <VersionPrefix>4.0.0</VersionPrefix>
     <Authors>Serilog Contributors</Authors>
-    <TargetFrameworks>net45;netstandard1.1;netstandard1.2;netstandard2.0;netstandard2.1</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT'">net462</TargetFrameworks>
+    <TargetFrameworks>$(TargetFrameworks);netstandard2.0;netstandard2.1;net6.0</TargetFrameworks>
```

Comment:
why retaining NS2.1? (Struggling to recall a good reason? https://github.com/serilog/serilog/issues/1970#issuecomment-1814286393)